### DISCUSSION TITLE
Dudas pregunta siete

### DISCUSSION THREAD

**Discussion by GerardoCabreraH** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#453] [Q&A] [answered]
Pues estaba verificando el formulario de la entrega y estaba viendo esta pregunta opcional sobre otras rutas que integre al API, que puede significar eso.
![pregunta](https://user-images.githubusercontent.com/99288879/159082962-4724c065-938e-4622-8b5b-4a99a2b0872b.png)

**Comment by chistoperez** [comment, marked as answer]
la api traía otras secciones aparte de `https://pokeapi.co/api/v2/pokemon/{id or name}` para sacar los nombres y sus características, estaba  `https://pokeapi.co/api/v2/evolution-chain/{id}/` para sacar cadenas evolutivas, `https://pokeapi.co/api/v2/move/{id or name}/` para movimientos, había otra para desplegar si un pokemon tenía diferentes formas, como alola o kanto o así. Creo que a eso se refiere con otras rutas

**Reply by GerardoCabreraH** [reply]
Muy bien. Muchas gracias.
